src/main/webapp/js/adminAccountManagement.js

```diff
@@ -15,7 +15,7 @@ $(document).ready(function() {
     showFirstPage();
     updateEntriesCount();
     bindDeleteAccountAction();
-    AdminCommon.bindBackToTopButtons();
+    AdminCommon.bindBackToTopButtons('.back-to-top-left, .back-to-top-right');
```

Comment:
Again, if this is something we want to add to all admin pages, we should add it to a common js that all admin pages have